packages/core/botpress/src/dialog/processors.js

```diff
@@ -20,11 +20,11 @@ module.exports = {
         })
       }
 
-      if (/^{.+}$/.test(rendered)) {
+      if (rendered && rendered[0] === '{' && rendered[rendered.length - 1] === '}') {
```

Comment:
I would be tempted to refactor this back to a Regex if I didn't know the context around performance costs. Could you add a comment saying that this is perf-wise much faster and to leave it as-is?

Reply:
Done